File: Client/src/content/life_2_tests.py
def run_tests() :
    fn = keep_within_bounds.__name__
    def test(p1, expected):
        test_function(fn, expected, keep_within_bounds(p1), [p1])
    test(399, 399)
    test(400, 0)
    test(401, 1)
    test(419, 19)
    test(0, 0)
    test(-1, 399)
    test(-20, 380)
    all_tests_passed()

File: Client/src/content/life_2_tests.cs
static void RunTests()
{
        testKeepWithinBounds(0, 0);
        testKeepWithinBounds(399, 399);
        testKeepWithinBounds(400, 0);
        testKeepWithinBounds(401, 1);
        testKeepWithinBounds(419, 19);
        testKeepWithinBounds(-1, 399);
        testKeepWithinBounds(-20, 380);
        AllTestsPassed();
}

private static void testKeepWithinBounds(int p1, int expected)
{
    TestFunction(nameof(KeepWithinBounds), expected, KeepWithinBounds(p1), p1);
}